apps/play/src/style.css

```
@@ -101,6 +101,8 @@ pan-drift-toggle {
 .page-wrapper {
   overflow-x: auto;
   overflow-y: auto;
+
+  margin: 0 auto;
```

Stylelint (17.13.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)